src/MusicStore/project.json

```diff
@@ -5,9 +5,6 @@
     "description": "Music store application on ASP.NET 5",
     "version": "1.0.0-*",
     "compilationOptions": { "warningsAsErrors": true, "define": [ "DEMO", "TESTING" ] },
-    "compile": [
-        "../../test/E2ETests/compiler/shared/**/*.cs"
-    ],
```

Comment:
There are no tests doing publish. The issue is that I'm not able to publish the tests on Nano server because of this